docs/index.html

```diff
@@ -11,7 +11,7 @@
     />
     <script src="https://kit.fontawesome.com/1f723026cf.js" crossorigin="anonymous"></script>
     <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.1/dist/js/bootstrap.bundle.min.js" integrity="sha384-gtEjrD/SeCtmISkJkNUaaKMoLD0//ElJ19smozuHV6z3Iehds+3Ulb9Bn9Plx0x4" crossorigin="anonymous"></script>
-
+    
```

Comment:
@twba-a did you manually change this file and re-save it? Wondering if these are caused by our touching the files or the build process...

Reply:
sorry if I wasn't clear in the description but this file is not touched. This file gets generated by **@11ty/eleventy**
its generated from nunjack index file [index.njk](https://github.com/talis/bootstrap-theme/blob/main/src/docs/index.njk#L15) .